docs/root/configuration/http/http_filters/ext_proc_filter.rst

```diff
@@ -17,3 +17,25 @@ messages, and the server must reply with
 :ref:`ProcessingResponse <envoy_v3_api_msg_service.ext_proc.v3alpha.ProcessingResponse>`.
 
 This filter is a work in progress. In its current state, it actually does nothing.
+
+Statistics
+----------
+This filter outputs statistics in the
+*http.<stat_prefix>.ext_proc.* namespace. The :ref:`stat prefix
+<envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.stat_prefix>`
+comes from the owning HTTP connection manager. Statistics are specific to the concurrency
+controllers.
```

Comment:
Not concurrency controllers!

Reply:
Oops. Took that part out and I think that the rest is accurate.